website/pages/docs/calendar/plugins/calendar-controls.mdx

```diff
@@ -18,6 +18,50 @@ Change the calendar view.
 
 Set the date of the calendar. Format: `YYYY-MM-DD`.
 
+### `setFirstDayOfWeek(firstDayOfWeek: WeekDay): void`
+
+Set the first day of the week for the calendar. Value must be between 0 and 6 where 0 is Sunday, 1 is Monday etc.
+
+### `setLocale(locale: string): void`
+
+Set the locale of the calendar.
+
+### `setDefaultView(defaultView: ViewName): void`
```

Comment:
I think this one doesn't make sense to add, since there already is the `setView` method

Reply:
Good point. I guess making `$app.config.defaultView` a signal doesn't make sense either then, will change